.azure-pipelines.yml

```diff
@@ -229,9 +247,6 @@ jobs:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: '3.6.x'
-  - task: UsePythonVersion@0
```

Comment:
I've asked about this in https://github.com/web-platform-tests/wpt/pull/20725/files#r377569491. It caused the job to start failing, so I'll probably revert and deal with it separately.